easybuild/easyconfigs/e/ELSI-RCI/ELSI-RCI-0.1.0-foss-2021a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'ELSI-RCI'
+version = '0.1.0'
+
+homepage = 'https://wordpress.elsi-interchange.org/'
+description = """ELSI-RCI provides and enhances open-source software packages
+which iteratively solve or circumvent eigenvalue problems in self-consistent
+field calculations based on the Kohn-Sham density-functional theory.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': False, 'pic': True}
```

Comment:
It does not look like ELSI-RCI uses a maths library (BLAS, MKL, etc.) or MPI. This means that we should use the `GCC` toolchain instead, which will result in renaming to `ELSI-RCI-0.1.0-GCC-10.3.0.eb`

So here:
```
toolchain = {'name': 'GCC', 'version': '10.3.0'}
toolchainopts = {'pic': True}
```

Reply:
@branfosj The project README says that BLAS/LAPACK are required. See here: https://gitlab.com/elsi_project/elsi_rci/-/blob/master/README.md

Reply:
The PDF documentation has more detail on this requirement (https://gitlab.com/elsi_project/elsi_rci/-/blob/master/doc/elsi_rci_manual.pdf - page 7):

> If ELSI-RCI test programs need to be built, then the following linear algebra libraries are required:
> BLAS, LAPACK

Looking at [test/CMakeLists.txt](https://gitlab.com/elsi_project/elsi_rci/-/blob/master/test/CMakeLists.txt) I can see that the tests require MKL (not just BLAS & LAPACK) and the [toolchains/intel.cmake](https://gitlab.com/elsi_project/elsi_rci/-/blob/master/toolchains/intel.cmake) show how they expect the libraries to be set. This is why the tests fail to compile. For the `foss` version because there is no MKL (missing include file `/include/mkl_dfti.f90`) and for the `intel` version because they do not use CMake to find the required MKL libraries (so the linking fails).

Reply:
Ah, okay. Thanks for looking at this more in detail. I just trusted the developers to write good documentation (I should know better...)

Then I think it's best to follow your suggestion and use the GCC/intel-compilers toolchains instead of foss/intel and ignore tests. Would you agree?

Reply:
Yes, I think that is best.

Reply:
Done.